jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaEntrypointConstructor.java

```diff
@@ -37,6 +38,15 @@
         mainClass);
   }
 
+  /**
+   * Constructs the container entrypoint for the gcr.io/distroless/jetty base image.
+   *
+   * @return ["java", "-jar", "/jetty/start.jar"]
```

Comment:
Might be good to add a link for easy reference:

`@see <a href="https://github.com/GoogleContainerTools/distroless/blob/master/java/jetty/BUILD">https://github.com/GoogleContainerTools/distroless/blob/master/java/jetty/BUILD</a>`

Reply:
And also a TODO to just set the CMD?

Reply:
I'm always confused when mixing CMD and ENTRYPOINT so not understanding the entire story, but I'll add a TODO to use CMD instead of ENTRYPOINT.

Reply:
Oh I meant like set the CMD for `/jetty/start.jar` and have ENTRYPOINT be inherited (`java -jar`), so that the container entrypoint becomes `java -jar /jetty/start.jar`.

Reply:
Oh, in that case, I think we don't need to set CMD either if we inherit both CMD and ENTRYPONIT. CMD is set with `/jetty/start.jar` and ENTRYPOINT `[/usr/bin/java, -jar]` in the base Jetty image.